autowriter/mingpt/utils.py

```diff
@@ -0,0 +1,103 @@
+
+import os
+import sys
+import json
+import random
+from ast import literal_eval
+
+import numpy as np
+import torch
+
+# -----------------------------------------------------------------------------
+
+def set_seed(seed):
```

Comment:
The `set_seed` function currently sets the seed for random, numpy, and torch. If other libraries (like tensorflow) are used in the future, their seeds won't be set. Consider making this function more generic so it can set the seed for any library. [medium]